### PR TITLE
feat(menu): open fullscreen with Ctrl+Cmd+F on Mac

### LINKED ISSUE #2050: Use Ctrl+Cmd+F as full screen shortcut on MacOS
__Is your feature request related to a problem? Please describe.__

I tried today to go full screen using a common shortcut `Ctrl+Cmd+F` (e.g. in Google Chrome, Finder, MS Edge). Unfortunately, it doesn't work in Camunda Modeler. As a side note, `F11` doesn't help either but it might be related to my machine.

__Describe the solution you'd like__

Use Ctrl+Cmd+F as full screen shortcut on MacOS

__Describe alternatives you've considered__

Leave it as it is.

__Additional context__

Nay.

### PR DESCRIPTION
**Pull request by barmac**
Closes #2050

__Definition of Done__

* [ ] corresponds to [the design principles](https://github.com/bpmn-io/design-principles)
* [ ] corresponds to [the code standards](https://github.com/bpmn-io/bpmn-js/blob/master/.github/CONTRIBUTING.md#creating-a-pull-request)
* [ ] passes Continuous Integration checks
* [ ] available as feature branch on GitHub
  * [ ] contains the cleaned up commit history
  * [ ] commit messages satisfy our [commit message guidelines](https://www.conventionalcommits.org/)
  * [ ] a single commit closes the issue via Closes #issuenr

### CODE REVIEW

File: app/lib/menu/mac-os/mac-menu-builder.js [hunk 1]
```diff
@@ -71,6 +71,16 @@ class MacMenuBuilder extends MenuBuilder {
     }));
   }
 
+  getWindowSubmenuTemplate() {
+    const submenuTemplate = super.getWindowSubmenuTemplate();
+
+    const fullScreenEntry = submenuTemplate.find(({ label }) => label === 'Fullscreen');
+
+    fullScreenEntry.accelerator = 'Ctrl+Cmd+F';
```
Comment (pinussilvestrus) [inline app/lib/menu/mac-os/mac-menu-builder.js:79]: Is this a macOS-only shortcut?

Comment (barmac) [reply app/lib/menu/mac-os/mac-menu-builder.js:79]: Yup. And it's common among MacOS apps: VS Code, Chrome, Firefox, Finder, Preview etc.

Comment (pinussilvestrus) [reply app/lib/menu/mac-os/mac-menu-builder.js:79]: Do we already have for other platforms, like `F11` for Windows? Or do we already have it?

**Review by pinussilvestrus — APPROVED** [review]
Works for me!

**Review by andreasgeier — APPROVED** [review]
I did not check the implementation but the feature makes sense.

**Pull request merged**